### PR TITLE
login_spec: Give delete of home project an user

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: src/api/spec/features/webui/login_spec.rb [hunk 1]
```diff
@@ -13,6 +14,7 @@
   end
 
   scenario 'login without home project shows a link to create it' do
+    login admin
```
Comment (bgeuken) [inline src/api/spec/features/webui/login_spec.rb]: Why is this needed (now)?:thinking:

Comment (coolo) [reply src/api/spec/features/webui/login_spec.rb]: https://github.com/openSUSE/open-build-service/pull/7550/files#diff-37dd923f1e6421bf4aa8375fbe11ad01R821

Comment (coolo) [reply src/api/spec/features/webui/login_spec.rb]: nobody can't delete nobody's home

Comment (bgeuken) [reply src/api/spec/features/webui/login_spec.rb]: I see. But the test where still passing. Either nobody can actually delete nobody's home, or the User#current id set from another test.

Comment (coolo) [reply src/api/spec/features/webui/login_spec.rb]: You wish the test suite was that meaningful - see line 818 of the same file. there is a complete block that is just sniped out in rspec test suite.

Comment (bgeuken) [reply src/api/spec/features/webui/login_spec.rb]: Just [ran into that](e773bfb9f24f3663edcd79be8b883c1f142b0d46) myself. Some of the specs are failing now when VCR is disabled:

```bash
  20) Bootstrap_Packages log download logfile succesfully
      Failure/Error: raise ArgumentError, 'Requiring user, but found nobody' unless session
      
      ArgumentError:
        Requiring user, but found nobody
      # ./app/models/user.rb:228:in `session!'
      # ./app/models/package.rb:795:in `write_to_backend'
      # ./spec/bootstrap/features/webui/packages_spec.rb:21:in `block (2 levels) in <top (required)>'

```

Comment (coolo) [reply src/api/spec/features/webui/login_spec.rb]: see #7592 and #7581
